### PR TITLE
fix: annotations on parameters of unbound actions

### PR DESCRIPTION
**Pull request by drktfl**
This should now resolve annotations on action (and function) parameters, also for unbound ones.

Action parameter annotations are uniquely resolved since an action is either uniquely bound to an entity type or the single unbound action of the specified name.

This might be different for functions because they can be overloaded not only with respect to their bound entity type, but to all parameters, which could make resolution ambiguous. The first matching function is used in this case.

### CODE REVIEW

File: packages/annotation-converter/test/converterTest.spec.ts [hunk 1]
```diff
@@ -865,27 +865,27 @@ describe('Annotation Converter', () => {
...
             expect(result).toBeDefined();
             return result;
         }
 
         const dataFields1 = convertedTypes.entityTypes[0]?.annotations.UI?.LineItem as any[];
         expect(dataFields1[0].ActionTarget).toBe(getAction('TestService.action(TestService.Entity1)'));
         expect(dataFields1[1].ActionTarget).toBe(getAction('TestService.function(TestService.Entity1)'));
-        expect(dataFields1[2].ActionTarget).toBe(getAction('TestService.action'));
-        expect(dataFields1[3].ActionTarget).toBe(getAction('TestService.function'));
+        expect(dataFields1[2].ActionTarget).toBe(getAction('TestService.action()'));
```
Comment (nlunets) [inline packages/annotation-converter/test/converterTest.spec.ts:876]: Shouldn't the non () syntax still work ?

**Review by nlunets — COMMENTED** [review]
Looks good, but it is a minor change indeed, i don't think that it would break any functionality but we never know

**Review by nlunets — COMMENTED** [review]
Looks good, but it is a minor change indeed, i don't think that it would break any functionality but we never know

File: packages/annotation-converter/test/converterTest.spec.ts (hunk 1 above)
Comment (drktfl) [reply packages/annotation-converter/test/converterTest.spec.ts:876]: this should be covered by https://github.com/SAP/open-ux-odata/blob/01bcc25246419df9cf3bbe160ef04efe9087af1a/packages/annotation-converter/test/converterTest.spec.ts#L992-L1032

Comment (nlunets) [reply packages/annotation-converter/test/converterTest.spec.ts:876]: Ok then

**Review by nlunets — APPROVED** [review]

**Pull request merged**